docs/howto/conditional.adoc

```diff
@@ -0,0 +1,65 @@
+= How to perform actions based on previous command results
+
+qDup commands under a `then` will run if the parent command ran successfully and will run on the output of the parent command. For example:
+```yaml
+#...
+- sh: date
+  then:
+  - regex: Fri
+    then:
+    - log: "Pizza Friday!"
+#...
+```
+The `regex` command runs with input from the parent `sh` command and then `log` command runs if the `regex` successfully matched. If we ran with `check-exit-code` then the `regex` would only run if the `date` exit code was zero.
+
+== if statement
+qDup does not have an `if` command but we can create a similar behaviour with link:../reference/command/readstate.adoc:[`read-state`] or link:../reference/command/js.adoc[`js`].
```

Comment:
Can you explicitly document the use case 'I want to decide the if/else branch based on exit status of previous command'? Even if it is a recommendation to place a multi-line piece of bash script.

Reply:
I think exit codes deserves a separate how to. qDup was created to follow the same workflow we do at a terminal and I don't recall checking exit codes when working in an ssh terminal but that is still something we should discuss for users.

Reply:
I may have formulated that too generically. Suppose that on command line you would decide by typing `ls` and see if the file is in the list. You would just eyeball the result. When automating this in qDup you wouldn't do `sh: ls -> regex: ^filename$`, it's easier to just `[ -f filename ]`. Having the whole branch in a one big blob of bash script is possible, but it's not the most convenient thing to do as you lose syntax highlighting and line info in case of error. Extracting the script to `assets/` is doable, too, but you need to `upload` the script first.